docs/reference/requirement-sets/ribbon-api-requirement-sets.md

```diff
@@ -10,7 +10,10 @@ ms.localizationpriority: medium
 
 Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office application supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](../../develop/office-versions-and-requirement-sets.md).
 
-The Ribbon API set supports programmatic control of when custom Add-in Commands (that is, custom ribbon buttons and menu items) are enabled and disabled.
+The Ribbon API set supports programmatic control of when custom Add-in Commands (that is, custom ribbon buttons and menu items) are enabled and disabled and when contextual tabs appear on the ribbon.
```

Comment:
It looks like don't style "add-in commands" as a proper noun elsewhere in our docs (ex: [Add-in commands for Excel, PowerPoint, and Word](https://docs.microsoft.com/en-us/office/dev/add-ins/design/add-in-commands)
```suggestion
The Ribbon API set supports programmatic control of when custom add-in commands (that is, custom ribbon buttons and menu items) are enabled and disabled and when contextual tabs appear on the ribbon.
```